src/Features/CSharp/Portable/Completion/CompletionProviders/DeclarationNameCompletionProvider.DeclarationInfo.cs

```diff
@@ -251,13 +252,31 @@ internal struct NameDeclarationInfo
                 int position, CancellationToken cancellationToken, out NameDeclarationInfo result)
             {
                 result = IsFollowingTypeOrComma<VariableDeclarationSyntax>(token, semanticModel,
-                     v => v.Type,
-                     v => v.Parent is LocalDeclarationStatementSyntax l ? l.Modifiers : default(SyntaxTokenList?),
+                     typeSyntaxGetter: v => v.Type,
+                     modifierGetter: v =>
```

Comment:
returning default or null to pass if code is a bit opaque. can we add some comments saying we are returning default so that we can get pass that condition?

```csharp
                var modifiers = modifierGetter(target);
                if (modifiers == null)
                {
                    return default;
                }
```

Reply:
sorry i don't understand... where do you want me to add a comment?

Reply:
when i saw this code, I got confused why you are returning default for certain cases and null for other cases. and realized later you are doing this to pass that condition I pasted above. 

you can add delegate bool TryGetModifiers(SyntaxNode target, out SyntaxTokenList modifiers); and pass in (SyntaxNode target, out SyntaxTokenList modifiers) => ....

but I think that is just overkill. so asked to just add comment so that people know why you do this return default or null thing.

Reply:
fixed in ad770dce343a57fa62c75368be125791cdc21ffb